front/src/modules/ui/table/components/EntityTableHeader.tsx

```diff
@@ -135,6 +163,29 @@ export function EntityTableHeader() {
     onMouseUp: handleResizeHandlerEnd,
   });
 
+  const toggleColumnMenu = useCallback(() => {
+    setIsColumnMenuOpen((previousValue) => !previousValue);
+  }, []);
+
+  const handleAddViewField = useCallback(
```

Comment:
LGTM, but I feel we will need to extract that to a hook in the next PR as we will be able to add viewFields from two different places (Options menu and + icon in the table header)